MediaManager/Plugin.MediaManager.Abstractions/Implementations/MediaFile.cs

```diff
@@ -17,12 +17,15 @@ public MediaFile(string url, MediaFileType type)
 
         public Guid Id { get; set; }
 
+        public string Artist { get; set; }
```

Comment:
Do we actually need this here if we have an implementation of `IMediaFileMetadata`?